changelogs/fragments/1331-proxmox-info-modules.yml

```
@@ -0,0 +1,7 @@
+---
+minor_changes:
+  - proxmox - create a shared doc_fragments (https://github.com/ansible-collections/community.general/pull/1331)
+  - proxmox - create a shared module_utils (https://github.com/ansible-collections/community.general/pull/1331)
```

```suggestion
  - proxmox - extract common code and documentation (https://github.com/ansible-collections/community.general/pull/1331).
```
Users don't need to know what module_utils and doc_fragments are, and experts can look at the provided link :)